tap_tool/select.py

```diff
@@ -36,6 +36,27 @@ def select_integration(integrations, account_id, all_integrations):
         custom_error(f"Integrations are not selected. Aborting...", True)
 
     return {
-        integration: integrations_records[integration]["secret"]
+        integration: (
+            integrations_records[integration]["secret"],
+            get_ad_accounts(integrations_records[integration]),
+        )
         for integration in integrations
     }
+
+
+def get_ad_accounts(record):
+    accounts = []
+    config = record.get("config", None)
+    if not config:
+        return accounts
+    ad_accounts = config.get("ad_accounts")
```

Comment:
```suggestion
    ad_accounts = config.get("ad_accounts", [])
```
So that it won't fail if there is no `ad_accounts` in config